rero_ils/modules/ebooks/utils.py

```diff
@@ -149,14 +153,13 @@ def update_document_holding(record, pid):
                         holdings_type='electronic'
                     )
 
-    source_uris = [harvested_source.get('uri')
-                   for harvested_source in harvested_sources
-                   if 'source' in harvested_source]
-
     for holding_pid in Holding.get_holdings_pid_by_document_pid(pid):
         holding = Holding.get_record_by_pid(holding_pid)
         for electronic_location in holding.get('electronic_location', []):
             if electronic_location.get('source') and \
```

Comment:
When updating, we should also delete holdings if their location is not `online` anymore.

Reply:
Should be done during update of the library.